src/clr-angular/popover/tooltip/tooltip-content.spec.ts

```diff
@@ -1,67 +1,71 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
 import { Component } from '@angular/core';
 
-import { TestContext } from '../../data/datagrid/helpers.spec';
 import { IfOpenService } from '../../utils/conditional/if-open.service';
 import { Point } from '../common/popover';
 
 import { ClrTooltipContent } from './tooltip-content';
+import { spec, TestContext } from '../../utils/testing/helpers.spec';
+import { ClrTooltipModule } from './tooltip.module';
+import { UNIQUE_ID_PROVIDER } from '../../utils/id-generator/id-generator.service';
+
+interface TooltipContext extends TestContext<ClrTooltipContent, SimpleTest> {
+  ifOpenService: IfOpenService;
+}
 
 export default function(): void {
-  // FIXME: this doesn't even run yet, we don't have an all.spec for tooltips.
   describe('TooltipContent component', function() {
-    let context: TestContext<ClrTooltipContent, SimpleTest>;
+    spec(ClrTooltipContent, SimpleTest, ClrTooltipModule, { providers: [IfOpenService, UNIQUE_ID_PROVIDER] });
 
-    beforeEach(function() {
-      context = this.create(ClrTooltipContent, SimpleTest, [IfOpenService]);
-      context.getClarityProvider(IfOpenService).open = true;
-      context.detectChanges();
+    beforeEach(function(this: TooltipContext) {
```

Comment:
Need a test for setting the `id` and `role` values?

Reply:
Added input for id. Will add tests that cover use cases.